frontend/packages/schema-editor/src/components/TypesInspector/TypeItem.tsx

```diff
@@ -1,12 +1,13 @@
 import React from 'react';
-import { extractNameFromPointer, UiSchemaNode } from '@altinn/schema-model';
+import { extractNameFromPointer, FieldType, setType, UiSchemaNode } from '@altinn/schema-model';
 import { CogIcon, FileJsonIcon } from '@navikt/aksel-icons';
 import classes from './TypeItem.module.css';
 import classNames from 'classnames';
 import * as testids from '../../../../../testing/testids';
 import { useDispatch } from 'react-redux';
 import { setSelectedId } from '../../features/editor/schemaEditorSlice';
 import { DragAndDropTree } from 'app-shared/components/DragAndDropTree';
+import { useSchemaEditorAppContext } from '@altinn/schema-editor/hooks/useSchemaEditorAppContext';
```

Comment:
We should use absolute path for imports within the same package 😄

Reply:
It is no longer exists here 😊